docs/src/TextField.doc.js

```diff
@@ -199,6 +205,53 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="tagsExample"
+    name="Example: Tags"
+    description={`You can include \`Tag\` elements in the input using the \`tags\` prop.`}
+    defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+  const [tags, setTags] = React.useState(['a@pinterest.com', 'b@pinterest.com']);
+  return (
+    <Box padding={2} color="white">
+      <TextField
+        id="emails"
+        label="Emails"
+        onChange={({value}) => {
+          const chunks = value.split(/[ ,;]+/);
+          if (chunks.length > 1) {
+            setTags([...tags, ...chunks.slice(0, -1)]);
+          }
+          setValue(chunks[chunks.length - 1]);
+        }}
+        onKeyDown={({event: {keyCode, target: {selectionEnd}}}) => {
```

Comment:
can we add spaces here? 

`        onKeyDown={({ event: { keyCode, target: { selectionEnd }}}) => {
`